rasgotransforms/rasgotransforms/snippets/alias.sql

```diff
@@ -1 +1 @@
-AS {{ alias }}
\ No newline at end of file
+as {{ alias }}
```

Comment:
files in the snippets folder shouldn't have a line break at end of the file. i'm guessing the new sqlformat hook is doing this? Any way we can exclude this folder from the formatting?

Reply:
Yeah I am pulling the formatter into it's own pr, so I reverted those changes. I'll be sure to exclude snippets in that one though.